README.md

```diff
@@ -101,11 +101,19 @@ For compression, transcoding and minification options, see `compy --help`
 Docker Usage
 ------------
 
-Andrew Gaul publishes unofficial Docker images at
-https://hub.docker.com/r/andrewgaul/compy/ .  You can configure via:
```

Comment:
Do we need to remove this?  Ideally casual users would not need to build the docker image.

Reply:
I can't push to it. I could set it up in my docker image repo if you like.